src/main/resources/assets/paths/userById.yaml

```diff
@@ -1,4 +1,5 @@
 put:
+  deprecated: true
```

Comment:
My Swagger page isn't showing this endpoint as deprecated for some reason

Reply:
Let me look into that - I don't think I checked for the deprecation in the ui.

Reply:
I'm seeing it as deprecated here:

<img width="703" alt="Screen Shot 2022-07-11 at 5 24 49 PM" src="https://user-images.githubusercontent.com/116679/178361172-d8364afb-fc16-442e-92cb-ee3b9c9c6052.png">